contrib/coredns/policy/policy.go

```diff
@@ -466,8 +481,14 @@ func (p *policyPlugin) ServeDNS(ctx context.Context, w dns.ResponseWriter, r *dn
 		sendExtra bool
 	)
 
-	debugQuery := p.decodeDebugMsg(r)
 	qName, qType := getNameType(r)
+	for _, s := range p.passthrough {
+		if strings.HasSuffix(qName, s) {
+			return plugin.NextOrFailure(p.Name(), p.next, ctx, w, r)
```

Comment:
Hm, what should we report with DNStap this case?

Reply:
I think no